src/handler.rs

```diff
@@ -84,6 +85,8 @@ struct GenerateMonitorAction {
     pipeline_out: String,
     #[clap(long, default_value_t = DEFAULT_PIPELINE_JOB_PREFIX.to_owned())]
     job_prefix: String,
+    #[clap(long, default_value_t = DEFAULT_PIPELINE_JOB_TIMEOUT.to_owned())]
```

Comment:
This shouldn't have a default set by us; If the user doesn't pass a timeout argument the generated job should simply not have a declared timeout which will cause it to inherit the timeout from the project configuration.  In hindsight we should have done the same for artifact expiry (so the default is the standard gitlb default) but that's orthogonal to this PR :)

Reply:
Default removed.